pages/agenda/2016.tsx

```diff
@@ -1,211 +1,225 @@
-                <td colSpan={3} className="breadth">
-                  <strong>Thank yous and wrap up</strong>
-                </td>
-              </tr>
+import { SponsorType, Session } from '../../config/types'
+import moment from 'moment'
+import { agendaPage } from '../../components/Agenda/AgendaPage'
+import { Agenda } from '../../components/Agenda/Agenda'
+import { AgendaProvider } from '../../components/Agenda/AgendaContext'
+import { From2019 } from '../../config/2019'
+import {
+  StyledAgendaRowList,
+  StyledAgendaRow,
+  StyledTrackHeader,
+  StyledAddress,
+} from '../../components/Agenda/Agenda.styled'
+import { AgendaTime } from '../../components/Agenda/AgendaTime'
+import { AgendaSession } from '../../components/Agenda/AgendaSession'
+import { StyledAgendaPresenter } from '../../components/Agenda/AgendaSession.styled'
+
+const agendaParams: AgendaPageParameters = {
+  conferenceInstance: '2016',
+  numTracks: 3,
+  sessionsUrl: '/static/agenda/2016.json',
+}
 
-              <tr className="breadth-row">
-                <td className="time">17:00</td>
-                <td colSpan={3} className="breadth">
-                  <strong>Afterparty</strong>
-                  <br />
-                  <em>My Place Bar & Restaurant</em>
-                  <br />
-                  70 Pier Street, Perth
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
-        <h2>Handbook</h2>
-        <p>
-          <a className="btn btn-pdf" href={From2016.HandbookUrl}>
-            Download 2015 Handbook (PDF)
-          </a>
-        </p>
-        <h2>Media</h2>
-        <div className="text-center">
-          <ResponsiveVideo src={From2016.YouTubeKeynoteEmbedUrl} />
-          <ResponsiveVideo src={From2016.YouTubeLocknoteEmbedUrl} />
-        </div>
-        <p>
-          <SafeLink href={From2016.YouTubePlaylistUrl} target="_blank">
-            YouTube Playlist
-          </SafeLink>{' '}
-          |{' '}
-          <SafeLink href={From2016.FlickrAlbumUrl} target="_blank">
-            Flickr Album
-          </SafeLink>
-        </p>
-        <Sponsors
-          show={true}
-          hideUpsell={true}
-          sponsors={From2016.Sponsors.filter(s => s.type === SponsorType.Gold || s.type === SponsorType.Platinum)}
-        />
-      </Fragment>
-    )
-  }
+const Agenda2016: React.FC<AgendaPageProps & { sessions: Session[] }> = ({ sessions }) => {
+  const date = moment.parseZone('2016-08-27T08:30:00+08:00')
+
+  return (
+    <Fragment>
+      <Agenda
+        sessions={sessions}
+        sessionsUrl={agendaParams.sessionsUrl}
+        acceptingFeedback={false}
+        selectedSessionId={undefined}
+        hideLevel={true}
+        hideTags={true}
+        render={(agendaSessions, _, onSelect) => {
+          return (
+            <AgendaProvider
+              onSelect={onSelect}
+              sessions={agendaSessions}
+              sponsors={From2019.Sponsors}
```

Comment:
Same here

Reply:
🤦‍♂ 

Have fixed both. Thanks for catching